src/program.js

```diff
@@ -108,7 +115,14 @@ export class Program {
       if (!runCommand) {
         throw new UsageError(`Unknown command: ${cmd}`);
       }
+
+      this.checkForUpdates ({
+        name: 'web-ext',
+        version: defaultVersionGetter(absolutePackageDir),
```

Comment:
this function already has a `getVersion` parameter which can be replaced by a stub for testing. You can use it like `checkForUpdates({version: getVersion(absolutePackageDir)})`